### DISCUSSION TITLE
Can't get to work on Chrome, Cypress 8.7.0

### DISCUSSION THREAD

**Discussion by agalias** [kamranayub/cypress-browser-permissions#38] [Q&A] [answered]
Hello, I encountered an issue where this plugin just doesn't work with provided instructions on the newer versions of Cypress Chrome. Could someone confirm that it works properly on the Cypress Chrome newer versions please?

**Comment by kamranayub** [comment, marked as answer]
This is working in my PR #37 with Chrome 95 and Cypress 9 (no changes to the underlying code, though). What issues are you running into?

update: Can you try with the latest version?
